bridge/client/app/project-board/project-board.component.ts

```diff
@@ -29,6 +29,7 @@ export class ProjectBoardComponent implements OnInit, OnDestroy {
     this.route.params
       .pipe(takeUntil(this.unsubscribe$))
       .subscribe(params => {
+        this.error = null;
```

Comment:
Maybe it would be better to put it as `else` statement at https://github.com/keptn/keptn/blob/a2dd0b1b971ccfaa8cac00e8fc9c3139c7ea7d32/bridge/client/app/project-board/project-board.component.ts#L66-L67, so in case that the "Retry"-Button for some reason once returns `null` and then later a `project`, the error is also set to `null`